### PR TITLE
tabbedAnchors-PTO-30

### PR DESCRIPTION
**Pull request by leejordan**
fixed this bug: https://naturedelivered.atlassian.net/browse/PTO-30

Is a blocker for shop development so would appreciate you looking at it ASAP so I can make a new release

### CODE REVIEW

File: js/src/modules/tabbed-content.js [hunk 1]
```diff
@@ -41,13 +41,14 @@ module.exports = function($) {
...
-                        if(shouldScrollTo) {
-                            $('body').stop().animate({ scrollTop: $elem.offset().top }, 200);
-                        }
                     } else {
                         $('.tabbed-content__section-wrap').append($(contentId));
                     }
                 }
+
+                if(shouldScrollTo) {
+                    $('body').stop().animate({ scrollTop: $elem.offset().top }, 300);
```
Comment (notlee) [inline js/src/modules/tabbed-content.js]: Looks like you've made the scroll-to animation slower. Just wanted to confirm that's intentional.

Comment (leejordan) [inline js/src/modules/tabbed-content.js]: Yea this was intentional - it feels a bit more natural and less sudden

**Comment by notlee** [comment]
@leejordan looks like when clicking an anchor link the back button would go back to pre-anchor which is kinda weird and confusing especially given the hash is removed. I've prevented that by going back on hash-change programatically. Feels a little gross. Think we should revisit this and see if it's possible to eliminate the need to remove the hash. Have a play around and check browser compatibility. IE9 is not great when using anchors.

**Comment by leejordan** [comment]
I've made some changes to the way this works and it updates the location hash properly now. I also found it works fine in ie9, although the offset doesn't scroll it to make the tab visible it still displays the content fine

**Comment by notlee** [comment]
As discussed... clicking wildly.
screen shot 2016-08-18 at 09 47 11
screen shot 2016-08-18 at 09 51 40

**Comment by notlee** [comment]
Cool! That does the trick. I actually prefer that the back button only effects normal anchor links elsewhere on the page and not when clicking through the tabs themselves. The scroll is pretty jarring on mobile though still, but I think it's always been like that, something we'll have to revisit. I'll add that to my list. 👍

**Pull request merged**